README.md

```diff
@@ -1,24 +1,71 @@
 # Environment-Setup
-Instructions on how to set up NHRAP HAZUS Open Source tool Environment
+Instructions on how to set up a Python environment to run NHRAP-HAZUS Open Source tools.
 
 ## Requirements
 
 Each NHRAP Hazus open source tool has their own requirements however each tool will require Python. ArcGIS Desktop includes Python 2.7 which is used to install conda. Some older tool versions require Anaconda be installed. Anaconda can be installed alongside Miniforge however if you do not have Anaconda installed then follow the Steps to Install Conda below.
 
 ## Documentation
+* [Conda](https://docs.conda.io/en/latest/) is a free software package manager that automatically manages all Python packages required to run Hazus open source tools.
+* [Miniforge](https://github.com/conda-forge/miniforge) is a community driven minimalistic conda installer.
 
-Conda is a free software package manager that automatically manages all Python packages required to run Hazus open source tools - including the Hazus Python package: https://fema-nhrap.s3.amazonaws.com/Hazus/Python/build/html/index.html - conda can be installed with Miniforge.]]
+## Steps to install conda using Miniforge
+1. Download this repository as a zipfile
+    ![download screenshot](Images/Step1.png "unzip screenshot")
+2. Unzip the download folder and run the python script install-miniforge.py by double-clicking it
+    ![unzip screenshot](Images/Step1.png "unzip screenshot")
+3. Download and run any of the NHRAP-HAZUS Open Source tools.
+    * Each tool has it's own environment.yaml file that conda will use to create a virtual environment with the required Python libraries.
 
+## Steps to uninstall conda 
+1. FIXME
```

Comment:
Why are the "FIXME" lines here? The README document doesn't seem complete & ready to merge with these here.

Reply:
Changed FIXME to Coming Soon. We'll want to add instructions at some point on how to uninstall conda and Miniforge.